Assets/Scripts/UI/PlayerHUDController.cs

```diff
@@ -5,6 +5,8 @@
 using UnityEngine.Serialization;
 using static GunStats;
 using System;
+using Org.BouncyCastle.Asn1.Cmp;
+using Unity.VisualScripting.Antlr3.Runtime.Misc;
```

Comment:
Dette skulle ikke vært importert?

Reply:
Cursed imports that I have no idea how got there